internal/gencli/cmd_file.go

```diff
@@ -398,6 +410,7 @@ var {{ $pollingCmdVar }} = &cobra.Command{
 			printMessage(resp)
 			return
 		}
+		{{ end }}
 
 		fmt.Println(fmt.Sprintf("Operation %s not done", op.Name()))
```

Comment:
I'm not sure that either the original code or the code that I added is determining this correctly.

From [this documentation](https://pkg.go.dev/cloud.google.com/go/longrunning#Operation.Poll), I think we should be checking `op.Done()` to determine that the operation is done.

WDYT?

Reply:
Yeah so the previous code was (improperly) depending on the presence of a response message (i.e. it is non-`nil`) to determine if it was done. Now that we want to handle google.protobuf.Empty where there will be no response message returned by the operation, we should use `op.Done()` to check. 

You are right.